oci/spec_opts.go

```diff
@@ -190,7 +190,10 @@ func WithEnv(environmentVariables []string) SpecOpts {
 // WithDefaultPathEnv sets the $PATH environment variable to the
 // default PATH defined in this package.
 func WithDefaultPathEnv(_ context.Context, _ Client, _ *containers.Container, s *Spec) error {
```

Comment:
If this function is not intended to be used on windows, we should probably move this out to a new spec_opts_nonwindows.go file an override this function to return nil/unsupported in spec_opts_windows.go